### ISSUE TITLE
Image Picker - A screen reader doesn't announce the Add New item button

### ISSUE THREAD

**Issue by SjsJane** [surveyjs/survey-creator#5449]
T17932 - [Accessibility] Navigation - Image Picker - The image field is drag and drop only, with no file selection available.  
https://surveyjs.answerdesk.io/internal/ticket/details/T17932
***
To reproduce the issue, navigate to the add new item button. In the NVDA speech viewer, you'll see that the button doesn't have a meaningful caption.
![image](https://github.com/surveyjs/survey-creator/assets/168413195/72c0470b-ac3a-446a-8f71-40e98b23f04b)

Expected result: Avoice over should tell a user that clicking on the image upload button opens the file explorer using which they can select an image to upload.
